overlord/ifacestate/handlers.go

```diff
@@ -165,7 +165,7 @@ func (m *InterfaceManager) doRemoveProfiles(task *state.Task, tomb *tomb.Tomb) e
 	if err != nil {
 		return err
 	}
-	snapName := snapSetup.Name()
+	snapName := snapSetup.InstanceName()
```

Comment:
should we start renaming this kind of local vars to instanceName, later, or keep them snapName ?

Reply:
I can do some drive by changes, but rather in a separate PR carrying the bulk of changes in `overlord/snapstate`